contracts/oracles/sources/oracle.move

```diff
@@ -0,0 +1,65 @@
+/// Oracles package is responsible for live price of sui token
```

Comment:
This package is not just made for the SUI token price. Some other tokens can also be included in it in the future. Change this to be:

`Oracles module is responsible for fetching various token prices via multiple oracle services`